shims/common/src/main/scala/org/apache/gluten/config/GlutenConfig.scala

```diff
@@ -1738,4 +1738,11 @@ object GlutenConfig {
       .doc("Enable or disable columnar collectTail.")
       .booleanConf
       .createWithDefault(true)
+
+  val MEMORY_DUMP_ON_EXIT =
+    buildConf("spark.gluten.monitor.memoryDumpOnExit")
+      .doc("Whether to trigger native memory dump when executor exits.")
+      .booleanConf
+      .createWithDefault(false)
```

Comment:
Can we move this config into VeloxConfig.scala? It's just applicable to Velox backend. Please also document that to enable this feature, it also requires --enable_jemalloc_stats is set to ON when building Gluten native code.